src/webpage.cpp

```diff
@@ -1057,18 +1058,23 @@ QString WebPage::renderBase64(const QByteArray& format)
     return "";
 }
 
-QImage WebPage::renderImage()
+QImage WebPage::renderImage(bool onlyViewport)
```

Comment:
This is a no no. See http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html for details.